synapse/models/economic.py

```diff
@@ -23,6 +23,11 @@ def getModelDefs(self):
                     'doc': 'An Issuer Id Number (IIN).'}),
 
                 ('econ:pay:card', ('guid', {}), {
+
+                    'interfaces': ('econ:pay:instrument',),
```

Comment:
We can remove the `:contact` prop on `econ:pay:card` as well now that it is getting it from the interface.

Reply:
good catch, thanks!